Library/ViewModels/PledgeViewModel.swift

```diff
@@ -23,7 +23,7 @@ public protocol PledgeViewModelInputs {
     paymentData: (displayName: String?, network: String?, transactionIdentifier: String)
   )
   func paymentAuthorizationViewControllerDidFinish()
-  func pledgeAmountDidUpdate(to amount: Double)
+  func pledgeAmountDidUpdate(_ amount: (value: Double, isValid: Bool))
```

Comment:
Should we also update the parameter name here from `amount` to maybe `value`?

Reply:
Yeah, I'd be happy to...any ideas what to replace `value` to?

`pledgeAmountDidUPdate(_ value: (???: Double, isValid: Bool))`

Reply:
I suppose if we introduced a typealias like:
```swift
typealias PledgeAmountData = (amount: Double, isValid: Bool)
```
Then that would allow us to change the delegate to something like:
```swift
func pledgeAmountViewController(
  _ viewController: PledgeAmountViewController,
  didUpdateWith data: PledgeAmountData
)
```
Then it'd also help name our VM input on `PledgeViewModel`:
```swift
func pledgeAmountViewControllerDidUpdate(with data: PledgeAmountData)
```